.github/workflows/repo-validation-checks.yml

```diff
@@ -0,0 +1,71 @@
+name: "repo-validation"
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: validating action repos
+      id: checks_validation
+      uses: GH-ACE/action-repo-validation@releases/validation-dashboard
+      continue-on-error: true
+      with:
+        GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+        repositories: 'login,webapps-deploy,aci-deploy,build-vm-image,functions-action,functions-container-action,mysql,postgresql,powershell,sql-action,appservice-settings,variable-substitution,policy-compliance-scan,k8s-create-secret,k8s-deploy,k8s-lint,manage-azure-policy,setup-kubectl,aks-set-context,k8s-bake,static-web-apps-deploy,get-keyvault-secrets,AppConfiguration-Sync,k8s-set-context,setup-helm,container-scan,docker-login,publish-security-assessments,arm-deploy,cli'
+    - name: create-json
+      id: create-json
+      uses: jsdaniell/create-json@1.1.2
+      with:
+        name: "validation-dashboard/result.json"
+        json: "${{steps.checks_validation.outputs.validationResult}}"
+    - name: validation result and  Commit file
+      id: commit1
+      continue-on-error: true
+      run: |
+           git config --local user.email "balaga-gayatri@github.com"
```

Comment:
let's use the standard config credentials that we use for other workflows?

Reply:
Done !